apps/static/dance/ai/model/cached-spacy-palette-map.json

```diff
@@ -0,0 +1,67 @@
+{

```

Comment:
What does "spacy" indicate in this file name?

Reply:
SpaCy is the python library/model we used to generate these so I named them that although I totally get why that might be confusing